app/helpers/timelines_helper.rb

```diff
@@ -182,4 +182,37 @@ def push_timeline_options(timeline, target = gon)
   def timeline_options
     OpenStruct.new @timeline.options
   end
+
+  def new_timeline_link(project, &_block)
+    link_to({ controller: '/timelines', action: 'new', project_id: project },
+            title: l('timelines.new_timeline'),
+            class: 'button -alt-highlight') do
+      yield if block_given?
+    end
```

Comment:
Looking at all the occurences you are always doing a `yield if block_given?`, so this might be a good idea...